examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -238,6 +238,7 @@ func discoverLinks() ui.Element {
 - [Docs/ Tutorials](https://github.com/gnolang)
 - [Gno by example](https://gno-by-example.com/)
 - [Getting started video (soon)](#)
+- [Gno Playground](https://play.gno.land)
```

Comment:
We can move it up in the list. 3rd place looks ok.

Reply:
fixed